demo/db.py

```diff
@@ -9,7 +9,6 @@
     sa.Column('id', sa.Integer, nullable=False),
     sa.Column('login', sa.String(256), nullable=False),
     sa.Column('passwd', sa.String(256), nullable=False),
-    sa.Column('salt', sa.String(256), nullable=False),
```

Comment:
I insist that password should be salted.
But I'm ok with storing the salt inside `passwd` field, http://pythonhosted.org/passlib/ is the best way to deal with passwords.
BTW where we do password check?